### PR TITLE
Release v0.10.0

### PR DESCRIPTION
**Pull request by farquet**
Blog announcement and version bump

### CODE REVIEW

**Comment by axel22** [comment]
Some other resources hardcode the `0.9.0` version, currently - for example, the Travis CI definition.
I suggest that you do a grep to find all such occurrences, and update them.

**Comment by farquet** [comment]
> Some other resources hardcode the 0.9.0 version, currently - for example, the Travis CI definition.
I suggest that you do a grep to find all such occurrences, and update them.

Fixed.

**Comment by farquet** [comment]
Interesting. Updating the readme from within sbt with `runMain org.renaissance.Launcher --readme` doesn't provide the same output as using the bundled jar : 
```
diff --git a/README.md b/README.md
index 3aad9d2..038b28a 100644
--- a/README.md
+++ b/README.md
@@ -47,7 +47,7 @@ For example, you can specify `scala-kmeans` as the benchmark.
 The following is a complete list of command-line options.

-renaissance-core, version 0.10.0
+Renaissance Benchmark Suite, version 0.10.0
 Usage: renaissance [options] [benchmark-specification]

   -h, --help               Prints this usage text.
```

File: .travis.yml [hunk 1]
```diff
@@ -15,20 +15,20 @@ refs:
       - 'mkdir -p "$HOME/.prebuilt"'
       - 'ls "$HOME/.prebuilt"'
       - 'rm -rf "$HOME/.prebuilt"/*.jar'
-      - 'cp target/renaissance-gpl-0.9.0.jar "$HOME/.prebuilt/"`git rev-parse HEAD`.jar'
```
Comment (axel22) [inline .travis.yml]: Opt: perhaps there is a way to add an env variable to this file, to have a single version declaration.
If you do, I would suggest adding a comment to `version.sbt`, with a reminder that `.travis.yml` should also be changed when changing `version.sbt`.

**Comment by axel22** [comment]
> Interesting. Updating the readme from within sbt with `runMain org.renaissance.Launcher --readme` doesn't provide the same output as using the fatjar :
> 

I *think* that you should change this line to change the title:

```
  val renaissanceTitle = classOf[RenaissanceBenchmark].getPackage.getSpecificationTitle
```

File: .travis.yml (hunk 1 above)
Comment (farquet) [reply .travis.yml]: That's a good idea. I'll do it.
Let's try to make the release process as easy as possible. Btw, coffee is on me for the one who will take care of #46 ;)

**Pull request merged**